lib/Crust/Request.pm6

```diff
@@ -83,10 +83,7 @@ method header(Str $name) {
 
 method content() {
     # TODO: we should support buffering in Crust layer
-    my $input = $!env<p6w.input>;
-    $input.seek(0, SeekFromBeginning); # rewind
-    my Blob $content = $input.slurp-rest(:bin);
-    return $content;
+    return await $!env<p6w.input>.Promise;
```

Comment:
Since [`Supply#Promise` will be resolved to the **last** emitted value](https://docs.perl6.org/type/Supply#method_Promise), and (in case of `HTTP::Server::Tiny`), `p6w.input` will be supplied as [`IO::Handle#Supply`, which may emit the contents in multiple chunks if the chunk is big enough.](https://docs.perl6.org/type/IO::Handle#method_Supply)

So, if the input is huge (> 65536 bytes), it seems this implementation cannot get whole of the input.

Like (in this example, size is reduced to 10 to demonstrate):
```
[github.com/tokuhirom/p6-Crust]$ perl6
To exit type 'exit' or '^D'
> my $supply = "README.md".IO.open(:bin).Supply(:size<10>);
Supply.new
> my $contents = await $supply.Promise;
Buf[uint8]:0x<65 20 32 2e 30 2e 0a>
> $contents.decode('ascii');
e 2.0.

>
[github.com/tokuhirom/p6-Crust]$ tail -1 README.md
This library is free software; you can redistribute it and/or modify it under the Artistic License 2.0.
```